packages/twenty-front/src/modules/object-record/record-inline-cell/components/RecordInlineCellContainer.tsx

```diff
@@ -120,7 +120,7 @@ export const RecordInlineCellContainer = () => {
           )}
           {showLabel && label && (
             <StyledLabelContainer width={labelWidth}>
-              <EllipsisDisplay maxWidth={labelWidth}>{label}</EllipsisDisplay>
+              <EllipsisTextWithTooltip text={label} labelId={labelId} />
```

Comment:
think you can use existing component `<OverflowingTextWithTooltip ... />`

Reply:
Yes I forgot we already have one !

Reply:
Let's use `OverflowingTextWithTooltip`then.